src/App.js

```diff
@@ -1,20 +1,37 @@
 import React, { useEffect, useState } from "react";
 import AddUser from "./components/AddUser";
 import User from "./components/User";
+import ProductList from './components/products/ProductList';
+import { Card } from 'semantic-ui-react';
 
 const App = () => {
   const [users, setUsers] = useState([]);
+  const [productList, setProductList] = useState();
   useEffect(() => {
     fetchData();
+    fetchProductData();
   }, []);
 
   const fetchData = async () => {
```

Comment:
Please change the name of the function to a meaningful one! 
Example: fetchUsers